internal/v2_devices.go

```diff
@@ -0,0 +1,19 @@
+package internal
+
+import "fmt"
+
+// TODO: should this live in package sync2?
```

Comment:
I think so: the sync3 tables are notionally dependent on sync2. (There aren't any foreign key constraints that express this, however.)

Reply:
Moved to the new migrations.go file.